src/components/home/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+
+import Button from "../base/button";
+import CreateTask from "../task/create-new";
+import TaskCard from "../task/existing-card";
+import Styles from "./index.module.scss";
+
+function HomeContainer() {
+  const tasks = useSelector((state) => state.todo.tasks);
+  const [isNewTaskRequested, setIsNewTaskRequested] = useState(false);
+
+  const handleSuccessfulTaskEntry = () => {
+    setIsNewTaskRequested(false);
+  };
+
+  return (
+    <div className={`container mx-auto ${Styles.homeWrapper}`}>
+      <h1>Add Tasks</h1>
+      <div className={`flex justify-between ${Styles.buttonContainer}`}>
+        <Button onButtonClick={() => setIsNewTaskRequested(true)}>
+          Create
+        </Button>
+
+        <div>
+          <Button>All</Button>
+          <Button>Incomplete</Button>
+          <Button>Complete</Button>
```

Comment:
Could we assign these inside a variable ? Then loop it here ?
```suggestion
          {taskStates.map......
          "<Button>{state}</Button>"
          .......
  },
          }
```

Reply:
Moving the button list inside a const will increase complexity will adding event listeners to each components & decrease code readability. Rather can we wrap these inside another component while we write any logic related to these buttons ? @sai-wtag 

 Thank you